src/raftstore/store/config.rs

```diff
@@ -37,7 +37,7 @@ const PD_STORE_HEARTBEAT_TICK_INTERVAL: u64 = 10000;
 const STORE_CAPACITY: u64 = u64::MAX;
 const DEFAULT_NOTIFY_CAPACITY: usize = 4096;
 const DEFAULT_MGR_GC_TICK_INTERVAL: u64 = 60000;
-const DEFAULT_SNAP_GC_TIMEOUT_SECS: u64 = 60 * 10;
+const DEFAULT_SNAP_GC_TIMEOUT_SECS: u64 = 4 * 60 * 60; // 4 hours
```

Comment:
It depends. If the snapshot file is opened, it usually can keep sending. Otherwise, it will report failure and leader has to generate the snapshot file again.